enhancements/20190919-OVN-Kubernetes.md

```diff
@@ -48,39 +48,50 @@ ovn-northd, which converts logical objects in to flows.
 The data store is a replicated database, with a traditional master-backup
 architecture.
 
-The node-level process (ovn-southd) is responsible for programming the
+The node-level process (ovn-controller) is responsible for programming the
```

Comment:
Actually this should be "is responsible for programming flows from the OVN southbound database into the vswitch on the local node".

Reply:
this change still needed